README.md

```diff
@@ -52,7 +204,33 @@ Pre-commit framework need to get initialized.
 task pre-commit:init
 ```
 
-## Configuration
+### Coverage report
+
+Run coverage report.
+
+```bash
+sh ./coverage_report.sh
+```
+
+```bash
+Name                                                                Stmts   Miss  Cover
+---------------------------------------------------------------------------------------
+template_python/template_python_package/template_python_module.py       6      0   100%
+template_python/unittest/template_python_test.py                       18      1    94%
```

Comment:
If the coverage report is needed, the path `template_python` would need to be updated

Reply:
don't need coverage report for now